floss/language/go/extract.py

```diff
@@ -50,6 +52,8 @@ def extract_go_strings(
         """
         alignment = 0x8
         fmt = "<II"
+
+        # See https://github.com/mandiant/flare-floss/issues/805#issuecomment-1590510957 for regex explanation
```

Comment:
Ideally, we have it inline so that we can easily see, ah these bytes are `cmp ecx, ...` when working on the code.
```
.text:004034EF 81 F9 00 00 40 00 cmp     ecx, 400000h
```

Let's work on the regex before though (see other discussion).